.gitignore

```diff
@@ -4,6 +4,8 @@
 *.dll
 *.so
 *.dylib
+.idea/
+
```

Comment:
this are local ignores only pertinent to your development environment, they are not project specific, so you should put them in `.git/info/exclude` or configure a [global .gitignore ](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)file in your system.

Reply:
we could be more relaxed on this but oK is valid point

Reply:
sorry, I didn't want to sound too strict; it's a nitpicky comment, of course, but I wanted to emphasize a good practice :wink: